web/src/styles/onboarding.css

```
@@ -2501,6 +2501,40 @@
+.ceo-scan-chip-row {
+  display: inline-flex;
+  align-items: center;
+  gap: 8px;
+}
+
+.ceo-scan-chip-reason {
+  margin: 0;
+  font-size: 12px;
+  color: var(--text-secondary);
+  line-height: 1.4;
+  word-break: break-word;
```

Stylelint (17.11.1)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)